components/helptext/index.css

```diff
@@ -112,36 +92,63 @@
 		padding-block-end: var(--mod-helptext-bottom-to-text, var(--spectrum-helptext-bottom-to-text));
-		line-height: var(--mod-helptext-line-height, var(--spectrum-line-height-100));
+		line-height: var(--mod-helptext-line-height, var(--spectrum-helptext-line-height));
+		font-family: var(--mod-helptext-font-family, var(--spectrum-helptext-font-family));
+		font-style: var(--mod-helptext-font-style, var(--spectrum-helptext-font-style));
+		font-weight: var(--mod-helptext-font-weight, var(--spectrum-helptext-font-weight));
 	}
+}
 
-	&.spectrum-HelpText--neutral {
-		.spectrum-HelpText-text {
-			color: var(--highcontrast-helptext-content-color-default, var(--mod-helptext-content-color-default, var(--spectrum-helptext-content-color-default)));
-		}
+/* Variants -- Neutral, negative, disabled */
+.spectrum-HelpText--neutral {
+	--spectrum-helptext-content-color-default: var(--spectrum-neutral-subdued-content-color-default);
+	--spectrum-helptext-icon-color-default: var(--spectrum-neutral-subdued-content-color-default);
 
-		.spectrum-HelpText-validationIcon {
-			color: var(--highcontrast-helptext-icon-color-default, var(--mod-helptext-icon-color-default, var(--spectrum-helptext-icon-color-default)));
-		}
+	.spectrum-HelpText-text {
+		color: var(--highcontrast-helptext-content-color-default, var(--mod-helptext-content-color-default, var(--spectrum-helptext-content-color-default)));
 	}
 
-	&.spectrum-HelpText--negative {
-		.spectrum-HelpText-text {
-			color: var(--highcontrast-helptext-content-color-default, var(--mod-helptext-content-color-default, var(--spectrum-helptext-content-color-default)));
-		}
+	.spectrum-HelpText-validationIcon {
+		color: var(--highcontrast-helptext-icon-color-default, var(--mod-helptext-icon-color-default, var(--spectrum-helptext-icon-color-default)));
+	}
+}
 
-		.spectrum-HelpText-validationIcon {
-			color: var(--highcontrast-helptext-icon-color-default, var(--mod-helptext-icon-color-default, var(--spectrum-helptext-icon-color-default)));
-		}
+.spectrum-HelpText--negative {
+	--spectrum-helptext-content-color-default: var(--spectrum-negative-color-900);
+	--spectrum-helptext-icon-color-default: var(--spectrum-negative-color-900);
+
+	.spectrum-HelpText-text {
+		color: var(--highcontrast-helptext-content-color-default, var(--mod-helptext-content-color-default, var(--spectrum-helptext-content-color-default)));
 	}
 
-	&.is-disabled {
-		.spectrum-HelpText-text {
-			color: var(--highcontrast-helptext-content-color-default, var(--mod-helptext-content-color-default, var(--spectrum-helptext-content-color-default)));
-		}
+	.spectrum-HelpText-validationIcon {
+		color: var(--highcontrast-helptext-icon-color-default, var(--mod-helptext-icon-color-default, var(--spectrum-helptext-icon-color-default)));
+	}
+}
+
+.is-disabled {
+	--spectrum-helptext-content-color-default: var(--spectrum-disabled-content-color);
+	--spectrum-helptext-icon-color-default: var(--spectrum-disabled-content-color);
```

Comment:
I think we've landed on the preference being to define and redefine all of the custom properties at the top of the file. So we might need to nest some small selector blocks within the first `.spectrum-HelpText` block with the rest of the tokens and custom properties. 

There's nothing wrong with the way this is, I just think we were trying to organize any custom property definitions at the top of the file instead.